src/lib/urn.ts

```diff
@@ -110,6 +110,17 @@ export function extractThirdPartyTokenId(urn: URN) {
   return `${thirdPartyCollectionId}:${thirdPartyTokenId}`
 }
 
+// TODO: This logic is repeated in collection/util's `getCollectionType`, but being used only for items (item.urn).
+// It should probably be replaced by a getItemType or we should see if it's better to only keep one way of doing this
+export function isThirdParty(urn?: string) {
```

Comment:
Could it be renamed to isThirdPartyItem then?
Can the urns be empty? Shouldn't we say that they could be strings or null values?

Reply:
> Could it be renamed to isThirdPartyItem then?

not really, it works for both types of URNs, Collections and Items. It is useful as is, but as I tried to say in the comment is not really being used right now for any other use case that for `item.urn`s

> Can the urns be empty? Shouldn't we say that they could be strings or null values?

I think this is well typed in item/types.ts where it says:

```ts
export type Item = BaseItem & {
  urn?: string
  //(...)
}
```

because in builder.ts we're doing the following:

```ts
// fromRemoteItem
  if (remoteItem.urn) item.urn = remoteItem.urn
```

right?

Reply:
The builder server should always return a valid URN (string) or a null value. This is done by using the `toFullItem` method.

Reply:
OH now I getcha, changing

Reply:
We should change the frontend too